Frontend/src/data/rental/onSave.js

```diff
@@ -76,9 +77,11 @@ export default async (rental, closePopup, updateItemStatus, createNew) => {
       `Did not update item of rental ${rental._id} because updateItemStatus is false.`
     );
   }
-
+ 
   await (createNew ? Database.createDoc(rental) : Database.updateDoc(rental))
     .then((result) => notifier.success("Leihvorgang gespeichert!"))
+    .then(()=>recentEmployeesStore.add(rental.passing_out_employee))
+    .then(()=>recentEmployeesStore.add(rental.receiving_employee))
```

Comment:
Here are some spaces missing. Could you format everything with prettier?

Reply:
whoops, it used to do this automatically